.configure

```diff
@@ -10,12 +10,12 @@
     },
     {
       "file": "shared/google_cloud_keys.json",
-      "destination": ".configure-files/google_cloud_keys.json",
+      "destination": "~/.configure/simplenote-macos/secrets/google_cloud_keys.json",
       "encrypt": true
     },
     {
       "file": "iOS/app_store_connect_fastlane_api_key.json",
-      "destination": ".configure-files/app_store_connect_fastlane_api_key.json",
+      "destination": "~/.configure/simplenote-macos/secrets/app_store_connect_fastlane_api_key.json",
```

Comment:
At some point in the future, we'll add some kind of `files_to_remove` annotation to clean up the devs' local repo. See also https://github.com/woocommerce/woocommerce-ios/pull/4576/files#r670733139.

Reply:
FYI, this idea is also tracked in https://github.com/Automattic/configure/issues/19

Reply:
Excellent, thanks!